Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+pipeline {
```

Comment:
I haven't tested this pipeline script yet, but I stole it from the pureport/kato-python-client project. 🤞